src/HDF5.jl

```diff
@@ -713,7 +713,7 @@ end
 function g_create(parent::Union{File,Group}, path::AbstractString,
                   lcpl::Properties=_link_properties(path),
                   dcpl::Properties=DEFAULT_PROPERTIES)
-    Group(h5g_create(checkvalid(parent), path, lcpl, dcpl), file(parent))
+    Group(h5g_create(checkvalid(parent), path, lcpl, dcpl, HDF5.H5P_DEFAULT), file(parent))
```

Comment:
`H5P_DEFAULT` (without `HDF5` prefix)

Reply:
Hah, probably ended up that way from reading the deprecation message and just doing as it told me to :-P